tailwind.config.js

```diff
@@ -2,7 +2,21 @@ module.exports = {
   purge: ['./src/pages/**/*.ts', './src/pages/**/*.tsx', './src/components/**/*.ts', './src/components/**/*.tsx'],
   darkMode: false, // or 'media' or 'class'
   theme: {
-    extend: {},
+    screens: {
+      sm: '375px',
+    },
+    colors: {
+      main: '#174DE5',
+      green: '#5CB940',
+      red: '#E51718',
+      white: '#FFF',
+      grey: {
+        light: '#F8F9FD',
+        DEFAULT: '#B4B4B4',
```

Comment:
Ref: https://tailwindcss.com/docs/customizing-colors#color-object-syntax

I saw the object syntax in the documentation. Does it support DEFAULT in colors object, not in each color?

Reply:
I think it supports DEFAULT. Because it works.

Reply:
Thanks